src/m365/teams/commands/meeting/meeting-create.ts

```diff
@@ -0,0 +1,206 @@
+import auth from '../../../../Auth.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import { Logger } from '../../../../cli/Logger.js';
+import { aadUser } from '../../../../utils/aadUser.js';
+import { accessToken } from '../../../../utils/accessToken.js';
+import { validation } from '../../../../utils/validation.js';
+import GraphCommand from "../../../base/GraphCommand.js";
+import commands from '../../commands.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { OnlineMeeting } from '@microsoft/microsoft-graph-types';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  startTime?: string;
+  endTime?: string;
+  subject?: string;
+  participants?: string;
+  organizerEmail?: string;
+  recordAutomatically?: boolean;
+}
+
+class TeamsMeetingCreateCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MEETING_CREATE;
+  }
+
+  public get description(): string {
+    return 'Create a new online meeting';
+  }
+
+  public defaultProperties(): string[] | undefined {
```

Comment:
We typically only use this on list commands. This is because a list with properties tends to be to wide for the terminal screen.

The current command is an add command, let's just remove the default properties function here. And don't forget to update the text response in the mdx file

Reply:
ok, I removed it, and updated all responses in mdx file